README.md

```diff
@@ -21,7 +21,7 @@ For a quick introduction to Joinmarket you can watch [this demonstration](https:
 * Can specify exact amount of coinjoin (figures from 0.01 to 30.0 btc and higher are practical), can choose time and number of counterparties
 * Can run passively to receive small payouts for taking part in coinjoins (see "Maker" and "yield-generator" in docs)
 * GUI to support Taker role, including tumbler/automated coinjoin sequence.
-* PayJoin - more economical and private payments between Joinmarket wallets.
+* PayJoin - [BIP78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) to pay others (e.g. merchants), and JM wallet to JM wallet payjoins also. This is a way to boost fungibility/privacy while paying.
```

Comment:
```suggestion
* PayJoin - Both [BIP78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) to pay users of other wallets, as well as between two JM wallet users. This is a way to boost fungibility/privacy while paying.
```

Reply:
It makes sense to specifically mention merchants imo, but agreed it should refer to cross-wallet compatibility (even if that's limited due to address type!).

Reply:
ACK - updated.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/633/commits/b5e87ba3052cf9bc85e016bfd582b70d7066eb98